docs/file-type-guidelines/compound-components.md

```diff
@@ -113,7 +113,7 @@ import { useSlotRecipe } from '@chakra-ui/react/styled-system';
 import { MenuTrigger as RaMenuTrigger } from 'react-aria-components';
 import { MenuRootSlot } from '../menu.slots';
 import type { MenuRootProps } from '../menu.types';
-import { extractStyleProps } from '@/utils/extractStyleProps';
+import { extractStyleProps } from "@/utils/extract-style-props";
```

Comment:
instead of importing from the `extract-style-props` file in `@utils`, you should import directly from `@utils`. That way if we refactor filenames/file structure within `/utils`, as long as the type is exported in `/utils/index.ts` the type will still be imported into wherever it's used without any need to make changes.
```suggestion
import { extractStyleProps } from "@/utils";
```